docs/src/main/asciidoc/transaction.adoc

```diff
@@ -5,13 +5,13 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 = Using Transactions in Quarkus
 include::_attributes.adoc[]
-:categories: data
-:summary: Quarkus comes with a Transaction Manager and uses it to coordinate and expose transactions to your applications. Each extension dealing with persistence will integrate with it for you, and you will explicitly interact with transactions via CDI. This guide will walk you through all that.
+:categories: data,transactions
 
-Quarkus comes with a Transaction Manager and uses it to coordinate and expose transactions to your applications.
-Each extension dealing with persistence will integrate with it for you.
-And you will explicitly interact with transactions via CDI.
-This guide will walk you through all that.
+Quarkus includes a Transaction Manager extension used to coordinate and expose transactions as described in the link: https://jakarta.ee/specifications/transactions/[Jakarta Transactions] specification, formerly known as Java Transaction API (JTA), to your applications.
+
+When discussing Quarkus transactions, this guide refers to Jakarta Transactions transaction style and thereafter uses the term _transaction_ to address them.
+
+NOTE: Quarkus does not support distributed transactions.
```

Comment:
Could you please expand this sentence? Michael's [comment](https://github.com/quarkusio/quarkus/pull/36179#issuecomment-1750293436) could be used to clarify what "distributed transactions" means here.